patch/series

```diff
@@ -193,9 +197,6 @@ cisco-hwmon-pmbus_core-pec-support-check.patch
 0001-hwmon-emc2305-Fix-unable-to-probe-emc2301-2-3.patch
 
 # sFlow + dropmon support
```

Comment:
Can you remove this line, since the patches have been moved up?

Reply:
Sure, will do